test/integration/022_bigquery_test/models/sql_header_model.sql

```diff
@@ -0,0 +1,8 @@
+{# This will fail if it is not extracted correctly #}
+{% call set_sql_header(config) %}
+  	create or replace table {{ ref('table_model') }} as (
```

Comment:
can you just make this a simple temporary UDF which is referenced in the model body below? I think that will more clearly exercise this new feature

Reply:
Definitely more clear. Good suggestion!